src/Symfony/Component/Form/Tests/Extension/Csrf/Type/FormTypeCsrfExtensionTest.php

```diff
@@ -161,7 +164,7 @@ public function testGenerateCsrfTokenUsesTypeClassAsIntentionIfEmptyFormName()
         $this->assertEquals('token', $view['csrf']->vars['value']);
     }
 
-    public static function provideBoolean()
+    public static function provideBoolean(): array
```

Comment:
Looks like there are also some unrelated change. Maybe it could be better to add those return type in a separate PR?